src/mixer.cr

```diff
@@ -40,17 +40,15 @@ module SDL
 
     # load long sound file
     def self.load_music(filename, type : Type? = nil)
-      #SDL::RWops.open(filename, "rb") do |rwops|
-      #  if type
-      #    audio = LibMixer.load_mus_type_rw(rwops, type.to_s, 1)
-      #    raise SDL::Error.new("Mix_LoadMUSType_RW") unless audio
-      #  else
-      #    audio = LibMixer.load_mus_rw(rwops, 1)
-      #    raise SDL::Error.new("Mix_LoadMUS_RW") unless audio
-      #  end
-      #  audio
-      #end
-      LibMixer.load_mus(filename)
+      rwops = LibSDL.rw_from_file(filename, "rb")
+      if type
+        audio = LibMixer.load_mus_type_rw(rwops, type.to_s, 1)
+        raise SDL::Error.new("Mix_LoadMUSType_RW") unless audio
+      else
+        audio = LibMixer.load_mus_rw(rwops, 1)
+        raise SDL::Error.new("Mix_LoadMUS_RW") unless audio
+      end
+      audio
```

Comment:
I like that. The only thing I can think of that we need to know weather a mus or wav is being loaded. Unless we say that the `Sample` class is specifically for loading a mus because of all the additional functions like stop, play, pause, etc... Then we would just leave the wav stuff as is.